offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/dataresources/persons/CustomerDbConnector.groovy

```diff
@@ -595,7 +594,7 @@ class CustomerDbConnector implements CreateCustomerDataSource, UpdateCustomerDat
    */
   List<Customer> fetchAllCustomers() {
     List<Customer> customers = []
-    String query = CUSTOMER_SELECT_QUERY
+    String query = CUSTOMER_SELECT_QUERY + " WHERE active=1"
```

Comment:
I think this change should be mentioned in the method description

Reply:
good point, so I will put that in the interface :+1:

Reply:
I actually ended up remove my change and use the other method that you introduced before.. I overlooked it.
b7422ab